docs/resources/ekscluster.md

```diff
@@ -260,6 +278,34 @@ Optional:
 - `security_groups` (Set of String) Security groups for the cluster VMs
 
 
+<a id="nestedblock--spec--config--addons_config"></a>
+### Nested Schema for `spec.config.addons_config`
+
+Optional:
+
+- `vpc_cni_config` (Block List, Max: 1) VPC CNI addon config contains the configuration for the VPC CNI addon of the cluster (see [below for nested schema](#nestedblock--spec--config--addons_config--vpc_cni_config))
```

Comment:
Lets add more details about vp cni config, eni config, the subnets, AZs and security groups in ENiConfigs, explaining the dependency of the AZ here with the AZs for Nodepools

Reply:
@DheerajSShetty  fixed this.